### DISCUSSION TITLE
抢cpp时候要关注支付界面

### DISCUSSION THREAD

**Discussion by mikumifa** [mikumifa/cppTickerBuy#37] [通知]
cpp因为服务器原因，可能已经抢到票了，但是当时返回报错，支付界面可能会过一会显示代付款，挂脚本时候记得关注支付界面

**Comment by kongfucurry** [comment]
确实，我本来都放弃了，回订单页一看，好家伙一个支付过期一个待支付，大佬牛波一

**Reply by kongfucurry** [reply]
小小star不成敬意

**Comment by Dilrevx** [comment]
我可能也是这种情况，还以为是手抢抢到了，xs

**Comment by BuCaicai** [comment]
大佬牛逼！我也是一个已经过期了，一个待支付的，哈哈哈

**Comment by 4675746SRDT** [comment]
痛失VIP

**Comment by MoonBerry175** [comment]
太感谢了太感谢了我本来开着挂后台当气氛组的没抱什么期望……伟大！！

**Comment by XuJ** [comment]
感谢大佬，我都是直到B站那边说同一身份证只能买一张，才反应过来CPP已经抢到了。估计是CPP的服务器接收到了请求，但是没返回

**Comment by P1KUMO** [comment]
哈哈哈痛失VIP，这服务器真的幽默，伟大！STAR！还好biliticket保了张普票
